src/SharpCompress/SharpCompress.csproj

```diff
@@ -35,7 +35,8 @@
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
     <PackageReference Include="System.Text.Encoding.CodePages" Version="5.0.0" />
-    <PackageReference Include="System.Memory" Version="4.5.4" />
+    <PackageReference Include="System.Memory" Version="4.5.3" />
+    <PackageReference Include="System.Buffers" Version="4.4.0" />
```

Comment:
Maybe a comment could be added that the `System.Buffers` package reference can be removed as soon as dotnet/runtime#1830 is fixed?